test/e2e/k8s_util.go

```diff
@@ -99,34 +100,35 @@ func (k *KubernetesUtils) probe(
 	cmd := []string{
 		"/bin/sh",
 		"-c",
+		fmt.Sprintf("for i in $(seq 1 3); do /agnhost connect %s:%d --timeout=1s --protocol=%s; done;", dstAddr, port, strings.ToLower(string(protocol))),
```

Comment:
I think you should have:
```go
protocolStr := map[v1.Protocol]string {
    {v1.ProtocolTCP, "tcp"},
    {v1.ProtocolUDP, "udp"},
    {v1.ProtocolSCTP, "sctp"},
}[protocol]
```
instead of relying on the string representation of `v1.Protocol`.

Reply:
Done.